src/BusinessLayer/SerialReporting.cpp

```diff
@@ -374,6 +369,27 @@ void SerialReporting::sendLights()
     communicationService_.sendSerialData(packet, packetLength);
 }
 
+void SerialReporting::sendAuxBms()
+{
+    const unsigned int unframedPacketLength = AUX_BMS_LENGTH + CHECKSUM_LENGTH;
+    unsigned char packetPayload[unframedPacketLength];
+    packetPayload[0] = 0x0C;
```

Comment:
I updated the protocol to have this be 11, instead of 12. Let's update that here